jhub_apps/service/japps_routes.py

```diff
@@ -0,0 +1,14 @@
+from fastapi import APIRouter, FastAPI, Request
+from fastapi.responses import HTMLResponse
+from fastapi.templating import Jinja2Templates
+
+app = FastAPI()
+templates = Jinja2Templates(directory="jhub_apps/templates")
```

Comment:
This path would not be accessible on the k8s environment, use this rather: https://github.com/nebari-dev/jhub-apps/blob/9d37c2f6e157758c0cd68d71a721521c5759bcc0/jhub_apps/__init__.py#L7

You may have seen that here: https://github.com/nebari-dev/jhub-apps/blob/9d37c2f6e157758c0cd68d71a721521c5759bcc0/jhub_apps/service/routes.py#L46

but that's a leftover from a cleanup and not used anywhere.

Reply:
Fixed